src/Utility/interface/ConfigMananger.h

```diff
@@ -5,8 +5,31 @@
 namespace Config
 {
 
+inline nlohmann::json genDefaultConfig()
+{
+  nlohmann::json cfg{};
+  nlohmann::json font = {};
+  std::string path;
```

Comment:
In theory, multiple font directories would be better. So this should be `std::vector<std::string>`